rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -125,7 +131,8 @@ def flush(self):
 
                     run_cmds.append("{} {} {}".format(cmd_prefix, ovn_cmd,
                                                       " ".join(self.cmds)))
-
+            if self.log_cmd:
+                LOG.info("".join(run_cmds))
```

Comment:
IIUC this should be `"\n".join(run_cmds)`.  This applies to multiple places in this PR.

Also, isn't it simpler if we create a base logging class in `ovsclients.py`?  Something like below:

```
from rally.common import logging

[...]

LOG = logging.getLogger(__name__)

def configure(name):
    return plugin.configure(name, namespace=_NAMESPACE)

class OvsClientLogger(object):
    def set_log_cmd(self, log_cmd=False):
        self.log_cmd = log_cmd

    def log_cmds(self, cmds):
        if self.log_cmd:
            LOG.info(cmds);
```

Then all client classes can inherit from this, i.e.:
```
class _OvnNbctl(DdCtlMixin, OvsClientLogger)
class _OvnSbctl(DdCtlMixin, OvsClientLogger)
class _OvsSsh(OvsClientLogger)
class _OvsVsctl(OvsClientLogger)
class _OvsOfctl(OvsClientLogger)
```